packages/netsuite-adapter/src/adapter.ts

```
@@ -317,10 +317,12 @@ export default class NetsuiteAdapter implements AdapterOperations {
       failedToFetchAllAtOnce, failedFilePaths, failedTypes, this.userConfig
     )
 
+    const errors = this.client.getErrors()
```

I find it a bit weird API that the client remembers the errors and then the adapter asks for it in the end.
I would expect the client to return the errors as part of the request that it serves.
i.e. `fetchByQuery` should return the errors.
WDYT?

---

makes sanse